pkg/webui/console/views/application-add/connect.js

```diff
@@ -32,6 +33,9 @@ const mapStateToProps = state => ({
   userId: selectUserId(state),
   rights: selectUserRights(state),
   fetching: selectOrganizationFetching(state),
+  jsConfig: selectJsConfig(),
+  nsConfig: selectNsConfig(),
+  asConfig: selectAsConfig(),
```

Comment:
You don't need to pass down environment as props here, you can just use the selectors directly within the application add component.